chains/solana/contracts/tests/ccip/ccip_router_test.go

```diff
@@ -2138,6 +2141,71 @@ func TestCCIPRouter(t *testing.T) {
 			require.NotNil(t, result)
 		})
 
+		t.Run("When sending with an empty but enabled allowlist, it fails", func(t *testing.T) {
```

Comment:
Should we add a test for sending with disabled allowlist as well?

Reply:
I don't think it's necessary: that's already captured by all the other tests, as the allowlist is disabled by default.